rep-0003.rst

```diff
@@ -308,6 +370,21 @@ ROS package downloads for this architecture.
 C++
 ---
 
+Melodic
+~~~~~~~
+As of ROS Melodic, we are using the C++14 (ISO/IEC 14882:2014) standard.  All
+packages are free to use C++14 features in public and private APIs.  Before
+changing existing public APIs to use C++14 features, package maintainers should
+carefully consider whether the change is worth the breakage to downstream
+consumers of the API.  If at all possible, maintainers should use a "tick-tock"
+model where the new APIs are introduced alongside the old (deprecated) APIs, and
+then remove the old APIs in a subsequent release.
+
+ROS Melodic is also compiler-agnostic, so no use of compiler-specific features is allowed
+without proper use of macros to allow use on other platforms.
```

Comment:
Does this apply only to ROS Melodic ?

Reply:
No, but there is similar wording in the "Lunar and earlier" section below.  I'll see about combining them so there is less duplication.